repos/fdbt-site/src/pages/api/defineCapStart.ts

```diff
@@ -0,0 +1,60 @@
+import { NextApiResponse } from 'next';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { CAP_START_ATTRIBUTE } from '../../constants/attributes';
+import { redirectToError, redirectTo } from '../../utils/apiUtils';
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { CapStartInfo, DayOfTheWeek } from '../../../src/interfaces/matchingJsonTypes';
+
+export const isADayOfTheWeek = (input: string): boolean => {
```

Comment:
This needs a unit test in the defineCapStart.test.ts file. Just needs a test to show it returns true when given a day, and false if given anything else.